config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: ghcr.io/open-feature/open-feature-operator
```

Comment:
Is this a regression? I thought we needed `newName` to be controller.

Reply:
Accidental, removed